src/logic/safe/transactions/gas.ts

```diff
@@ -92,6 +93,141 @@ export const estimateTxGasCosts = async (
   }
 }
 
+const getOpenEthereumErrorDataResult = (errorMessage: string): string | undefined => {
```

Comment:
does it make sense to move this function to a more generic place?

Reply:
I'm not sure because it will only be used here, do you have any ideas?